packages/eslint-plugin-wpcalypso/lib/configs/recommended.js

```diff
@@ -129,7 +129,6 @@ module.exports = {
 		'jsdoc/check-values': 'error',
 		'jsdoc/empty-tags': 'error',
 		'jsdoc/implements-on-classes': 'error',
-		'jsdoc/newline-after-description': 'error',
```

Comment:
This rule was broken and the recommendation is to have it disabled or removed. 

See https://github.com/gajus/eslint-plugin-jsdoc/issues/1031 for more details.